### ISSUE TITLE
pages index

### ISSUE THREAD

**Issue by hakase** [hakase/karasu#2]
Try and shift the logic into a macro so I'm not manually adding links in the jump nav or repeating code.

```
<h1>{{ this.title }}</h1>
<div class="nav">
  <ul class="inline-nav">
    <span class="alert">jump</span>
    <li><a href="#a">A</a></li>
    <li><a href="#b">B</a></li>
    <li><a href="#m">M</a></li>
    <li><a href="#o">O</a></li>
  </ul>
</div>
{% set pages = site.query('/')|sort(attribute='path') %}
{% set alphabet = [] %}
{% for page in pages %}
  {% set letter = page.title[:1]|lower %}
  {% if letter not in alphabet %}
    <h3 id="{{ letter }}">{{ letter|upper }}</h3>
    <ul class="index">
      {% set alphabet = alphabet + [letter] %}
      {% for page in pages %}
        {% if page.title[:1]|lower == letter %}
          <li><a href="{{ page.path }}">{{ page.title }}</a></li>
        {% endif %}
      {% endfor %}
    </ul>
  {% endif %}
{% endfor %}
```

**Comment by hakase** [comment]
When I've added more content... split it into two columns at higher breakpoints.

**Comment by hakase** [comment]
First part fixed in 4d0b059a23a03a3a90c4538544710ee054566957

**Comment by hakase** [comment]
https://www.fourkitchens.com/blog/article/responsive-multi-column-lists-flexbox

**Comment by hakase** [comment]
Fixed with feb8fc648c6372fbbf7b2edba7144005a8ceab7d
